src/models/event.js

```diff
@@ -12,4 +12,10 @@ const add = (type, userId, postId, referer, agent, ip, timestamp = new Date()) =
     .then(() => obj);
 };
 
-export default { getAll, add };
+const view = (userId, postId) => {
```

Comment:
One thing catches my attention here, this function is made especially for testing as it ignores the other parameters of an event. I'm afraid we can forget about it in the future and use it as is. What do you think?

Reply:
In that case, I think we can just use the add function and remove the view function. I just noticed that I used the add function for marking posts as read for the in the `post` test. https://github.com/dailydotdev/daily-api/blob/de492bd5e2d9b7d66cd5b4a442891885632f5c8a/test/integration/models/post.js#L175
We can use it in the graphql tests as well, as that's the only place we are using the view function right now.

Reply:
Ok so let's proceed with removing the function :)

Reply:
Done!

Reply:
Done @idoshamun